src/main/java/org/apache/mesos/specification/TaskSet.java

```diff
@@ -0,0 +1,11 @@
+package org.apache.mesos.specification;
+
+import java.util.List;
+
+/**
+ * A TaskSet groups a set of Tasks that should be deployed using the same deployment strategy.
```

Comment:
@nickbp and @mrbrowning:  Started looking at `PhaseSpecification` and it turns out to be identical to `TaskSet`.  So it's gone.